AwsCryptographicMaterialProviders/runtimes/python/.readthedocs.yaml

```diff
@@ -0,0 +1,37 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  jobs:
+    post_create_environment:
+      # Install poetry
+      # https://python-poetry.org/docs/#installing-manually
+      - pip install poetry
+      # Get Dafny.
+      # readthedocs executes each command in a new shell process, so exported variables aren't persisted between commands.
+      # Any command that relies on exported variables needs to be executed in one command.
+      - export dafnyVersion=$(grep '^dafnyVersion=' project.properties | cut -d '=' -f 2) && curl https://github.com/dafny-lang/dafny/releases/download/v$dafnyVersion/dafny-$dafnyVersion-x64-ubuntu-20.04.zip  -L -o dafny.zip
```

Comment:
nit: this seems awfully brittle, but i guess we do the same thing for Java :/

Reply:
Yeah. This is also not in a "critical" path since it's in the docs build.
(But we do the same thing in our release process, which is in a more critical path...)